### DISCUSSION TITLE
Folder/Dashboard user permission with wildcards

### DISCUSSION THREAD

**Discussion by fadeltd** [grafana/grafana#73188] [Dashboards, Panels & Visualizations]
Hi, I have an idea for user permission that currently matches my requirements.

<img width="1302" src="https://github.com/grafana/grafana/assets/4526413/c56e6d05-48b4-4021-9ffd-868f2c0b4b88">

Currently, for permission management, we need to add users one by one to dashboard permission, set up a team, and add the members there, or we need to add users one by one.

While for most parts, using `team` works, it can be quite time-consuming to add new members every time there are new members joining. So on our organization, we have a certain dashboard and folder that we want to restrict access to. As our organizations utilize a specific user account prefix/suffix or custom subdomains, it would be really helpful to have permission that utilize wildcards.

I'm suggesting a wildcard approach to the user permission settings. Instead of listing out each user separately, we could utilize wildcard patterns that match a group of users based on their common identifiers.

For example, consider the following scenario:

1. We have an admin team with members like `admin-team-1@domain.com` and `admin-team-2@domain.com`.
2. By implementing a wildcard pattern like `admin-*@domain.com`, we could grant the "admin" permission to all users in this group for a specific dashboard (let's say Dashboard A).

Similarly, for different organizations or subdomains, this wildcard approach could be just as useful. For instance:

1. In one organization with subdomains like `some.domain.com` and `othersub.domain.com`, we could grant view permissions for certain dashboards to all users under the `*@othersub.domain.com` wildcard pattern (Dashboard B).

I think having this feature can simplify access management to Grafana dashboards. Looking forward to your feedback.

Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82553. Feel free to continue the discussion around this there. Thank you!
